tldw_Server_API/app/api/v1/endpoints/prompt_studio/prompt_studio_websocket.py

```
@@ -394,6 +410,23 @@ async def sse_endpoint_route(
+    active_count=connection_manager.get_connection_count,
+    drain=connection_manager.close_all,
+)
+
+
+async def _guard_prompt_studio_websocket_start(websocket: WebSocket, kind: str) -> bool:
+    app = getattr(websocket, "app", None)
+    if app is None:
+        return True
+    try:
+        assert_may_start_work(app, kind)
+        return True
```

Ruff (0.15.7)

[warning] Consider moving this statement to an `else` block

(TRY300)